notifications.go

```diff
@@ -0,0 +1,271 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/go-telegram-bot-api/telegram-bot-api"
+	"io/ioutil"
+	"log"
+	"path/filepath"
+	"strconv"
+	"sync"
+)
+
+const (
+	notificationsDB = "notifications.json"
+)
+
+type notifications struct {
+	sync.RWMutex
+	Users map[string]string `json:"users"`
+}
+
+var (
+	userNotifications = notifications{Users: make(map[string]string)}
+)
+
+// notificationHandler enables/disables user notifications.
+func (x *opBot) notificationHandler(update tgbotapi.Update) error {
+	uidstr := fmt.Sprintf("%d", update.Message.From.ID)
+
+	if err := toggleNotifications(uidstr, update.Message.From.UserName); err != nil {
+		text := fmt.Sprintf("%s (%s)", T("notification_fail"), notificationStatus(uidstr))
+		x.sendReply(update, text)
+		log.Println(err)
+		return nil
+	}
+
+	text := fmt.Sprintf("%s (%s)", T("notification_success"), notificationStatus(uidstr))
+	x.sendReply(update, text)
+	return nil
+}
+
+// notificationStatus returns a string with the enabled/disabled status of notifications
```

Comment:
Format comments at 80 columns, if possible (I can  check it here, since github does not show me the line length), but it appears wrapped over 80 columns.